### ISSUE TITLE
Could not load file or assembly 'System.Drawing.Common'

### ISSUE THREAD

**Issue by nfriend** [thomasjo/highlight#5]
I get this error when calling `highlighter.Highlight("JavaScript", myString)`:

```
System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Drawing.Common, Version=0.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.'
```
I installed this library via Nuget.

I'm using .NET Core 2.1.302.  Is this library compatible with .NET Core?

**Comment by nfriend** [comment]
I was able to manually install `System.Drawing.Common` from Nuget, and this resolved the issue.  Perhaps this library needs to update its Nuget dependencies so that `System.Drawing.Common` is included by default?

**Comment by thomasjo** [comment]
This library was originally created about 14 years ago, and the last version of the .NET framework that I tested it with was v4.0. I suspect it won't be difficult to support .NET Core out of the box, but I don't do any .NET development these days and I no longer actively maintain this library.

I would be happy to accept a PR that resolves the issue though!

**Comment by thomasjo** [comment]
This should have been resolved by merging #9. If not, please leave a reply here with any necessary details about versions, platforms, and whatnot.

**Comment by philippjbauer** [comment]
Added PR https://github.com/thomasjo/highlight/pull/16. `System.Drawing` is only supported on Windows. This enables use with .NET on macOS / Linux systems.
